modules/apiserversource-kn.adoc

```diff
@@ -113,10 +136,34 @@ Data,
 [id="delete-apiserversource-kn_{context}"]
 == Deleting the ApiServerSource
 
-You can delete the ApiServerSource, trigger, service, service account, cluster role, and cluster binding created in this guide by entering the following `kn` and `oc` commands:
+. Delete the trigger by entering the following command:
```

Comment:
You can remove "by entering the following command" from each of these steps. For example, "Delete the trigger:"